dist/tools/doccheck/exclude_patterns

```diff
@@ -13110,3 +13110,41 @@ boards/sltb009a/include/periph_conf\.h:[0-9]+: warning: Member TIMER_[A-Z0-9_]+
 boards/sltb009a/include/periph_conf\.h:[0-9]+: warning: Member timer_config\[\] \(variable) of file periph_conf\.h is not documented.
 boards/sltb009a/include/periph_conf\.h:[0-9]+: warning: Member UART_[A-Z0-9_]+ \(macro definition) of file periph_conf\.h is not documented.
 boards/sltb009a/include/periph_conf\.h:[0-9]+: warning: Member uart_config\[\] \(variable) of file periph_conf\.h is not documented.
+boards/xg23b\-pk6068a/include/board\.h:[0-9]+: warning: Member CONFIG_ZTIMER_LPTIMER_DEV \(macro definition\) of file board\.h is not documented\.
```

Comment:
Number of entries could be reduced a lot by using regular expressions for identifiers, for example
```suggestion
boards/xg23b\-pk6068a/include/board\.h:[0-9]+: warning: Member CONFIG_ZTIMER_[A-Z_]+ \(macro definition\) of file board\.h is not documented\.
```

Reply:
5dfa95795953a61cc33281b095196a9ef776fc07